src/components/Collaborative/hooks/useUpdateSelection.tsx

```diff
@@ -0,0 +1,8 @@
+import { useCurrentSelection } from "@remirror/react";
+
+export default function useUpdateSelection() {
```

Comment:
question: this is returning a boolean right? If you're not using this on more than 1 file, maybe you should just do it in the code. The function name is confusing.